### ISSUE TITLE
Add rdatastore package

### ISSUE THREAD

**Issue by danielecook** [cloudyr/cloudyr.github.io#10]
I've written the following package:

https://github.com/danielecook/rdatastore

I think it would fit in with cloudyr. There is room for improvement on it (mainly using GoogleAuthR), but it is functional!

Let me know what you think!

I also had a suggestion for the website - I think it would be beneficial to have a navigational menu. Something like 'Home' -  'Packages' - 'Contributing', Just to make it a bit easier to find stuff?

**Comment by leeper** [comment]
Yes! This looks great. Definitely a good fit. I've sent you an invite to the cloudyr organization. Once you accept, I can make you a temporary admin and you can transfer the repo to the organization (from the repo settings page). I'll give it a look over and send a Pull Request with some cloudyr branding and style guide changes, and then you'll retain full control and maintainership from there. Sound good?

And good call on the website - it is unfortunately pretty low on my priority list, so if you have the skills and time, I would gladly accept any updates you might be able to contribute.

**Comment by danielecook** [comment]
Great. I'll clean up rdatastore and can move it over.

**Comment by leeper** [comment]
Hey, sorry, I lost track of this. I just gave you access to cloudyr, so that you can transfer the repo (from the repo's Settings page). I'll send the PR with branding stuff once that's done. Let me know if you have any challenges with the transfer.
